apps/docs/src/styles/adapter-ionic.scss

```
@@ -6,6 +6,13 @@
 @use 'ionic' as ember-ionic;
 @use 'examples' as examples;
 @use 'tokens' as *;
+// End users: `@use '@ng-forge/dynamic-forms-ionic/styles/defaults' as df;`
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)